tests/www/test_views.py

```diff
@@ -1922,6 +1922,24 @@ def test_rendered_view(self, get_dag_function):
         resp = self.client.get(url, follow_redirects=True)
         self.check_content_in_response("testdag__testtask__20200301", resp)
 
+    @mock.patch('airflow.www.views.STORE_SERIALIZED_DAGS', True)
+    @mock.patch('airflow.models.taskinstance.STORE_SERIALIZED_DAGS', True)
+    @mock.patch('airflow.www.views.dagbag.get_dag')
+    def test_rendered_view_for_unexecuted_tis(self, get_dag_function):
+        """
+        Test that the Rendered View is able to show rendered values
+        even for TIs that have not yet executed
+        """
+        get_dag_function.return_value = SerializedDagModel.get(self.dag.dag_id).dag
+
+        self.assertEqual(self.task.bash_command, '{{ task_instance_key_str }}')
+
+        url = ('rendered?task_id=testtask&dag_id=testdag&execution_date={}'
+               .format(self.percent_encode(self.default_date)))
+
+        resp = self.client.get(url, follow_redirects=True)
+        self.check_content_in_response("testdag__testtask__20200301", resp)
```

Comment:
So we attempt to render the fields anyway? Does this mean that using a user-defined filter etc will just display nothing? Error?

This is possibly more confusing than not rendering the result at all.

Reply:
It will error with the following message:

![image](https://user-images.githubusercontent.com/8811558/76248463-4811b780-6239-11ea-844f-f03a60d6b250.png)

Reply:
I am updating it and now it would show the following:

![image](https://user-images.githubusercontent.com/8811558/76253638-f40bd080-6242-11ea-86bb-246b77804ca4.png)

Reply:
Display original message too

Reply:
Nice -- lets display the original error too, as it could be a typo for a built-in filter, and that would help debug it.

Reply:
Updated, screenshot:

![image](https://user-images.githubusercontent.com/8811558/76525702-3d873600-6464-11ea-906d-9cf0d60b106a.png)